x/exchange/msg.go

```diff
@@ -462,14 +669,32 @@ func (m MsgGovManageFeesRequest) HasUpdates() bool {
 		len(m.AddFeeSellerSettlementFlat) > 0 || len(m.RemoveFeeSellerSettlementFlat) > 0 ||
 		len(m.AddFeeSellerSettlementRatios) > 0 || len(m.RemoveFeeSellerSettlementRatios) > 0 ||
 		len(m.AddFeeBuyerSettlementFlat) > 0 || len(m.RemoveFeeBuyerSettlementFlat) > 0 ||
-		len(m.AddFeeBuyerSettlementRatios) > 0 || len(m.RemoveFeeBuyerSettlementRatios) > 0
+		len(m.AddFeeBuyerSettlementRatios) > 0 || len(m.RemoveFeeBuyerSettlementRatios) > 0 ||
```

Comment:
Is there any way to make this more readable? There is a lot going on here in this single statement.

Reply:
It could be split up into multiple `if <thing> return true` statements, but all that would do is just spread things out (adding a couple lines between each individual check) making it harder to read.

Since each individual check is simple (either `len > 0` or straight boolean field), this is the best way I know of. It helps that each line associates two related fields (add and remove) too. It'd be helpful if `gofmt` allowed adding extra spaces in order to align the `||`, but we're stuck with what it is.

Reply:
Are there other solutions besides just multiple if statements? Can we break it up into variables that can shorten this and make it more readable?

`hasValidBuyerSettlementRatio := len(m.AddFeeBuyerSettlementRatios) > 0 || len(m.RemoveFeeBuyerSettlementRatios) > 0`

Reply:
Creating a bunch of variables like that would just obfuscate it further.